### PR TITLE
Mike

### PR DESCRIPTION
**Pull request by MichaelHofbeck**

### CODE REVIEW

File: neuralNet.py [hunk 1]
```diff
@@ -14,4 +15,170 @@
...
+optimizer = torch.optim.SGD(net.parameters(), lr=learning_rate, momentum=0.9, weight_decay=0.0001)
+
+"""
+Training the network for a given number of epochs
+The loss after every epoch is printed
+"""
+losses = []
+for epoch in range(num_epochs):
+    for idx, data in enumerate(train_loader, 0):
+        if epoch == num_epochs/2:   optimizer = torch.optim.SGD(net.parameters(), lr=learning_rate/100, momentum=0.2)
```
Comment (mccabe-david) [inline neuralNet.py:122]: I wonder if this actually works for adjusting learning rate later. Might want to use integer division instead...
